.circleci/config.yml

```diff
@@ -356,6 +356,21 @@ jobs:
             - gevent.2.results
       - *save_cache_step
 
+  influx:
+    docker:
+      - *test_runner
+      - image: influxdb:1.5-alpine
```

Comment:
Just noticed that the latest version is 1.6.x. Can we please pin `influxdb:1.6-alpine` here?